internal/requests/requests_test.go

```diff
@@ -110,27 +109,6 @@ func (suite *RequestsTestSuite) TestNewKosliClient() {
 	}
 }
 
-func (suite *RequestsTestSuite) TestNewStandardKosliClient() {
-	client, err := NewStandardKosliClient("")
-	require.NoError(suite.T(), err)
-	require.NotNil(suite.T(), client)
-
-	client.SetDebug(true)
-	require.True(suite.T(), client.Debug)
-
-	logger := logger.NewStandardLogger()
-	client.SetLogger(logger)
-	require.Equal(suite.T(), logger, client.Logger)
-
-	client.SetMaxAPIRetries(5)
-	require.Equal(suite.T(), 5, client.MaxAPIRetries)
-
-	client2, err := NewStandardKosliClient("http://192.0.0.1:8001")
```

Comment:
we may want to keep this portion of the test as it tests the http proxy is correctly set (unless there is another test doing the same)